setup.py

```diff
@@ -49,7 +49,7 @@
     "giturlparse",
     "sentry-sdk",
     "validators==0.28.3",
-    "jsonschema"
+    "jsonschema==3.2.0"
```

Comment:
you are sure this can not be looser?

Reply:
I will try

Reply:
without the restriction, I get this error (using python 3.10):

```bash
ERROR: flask-restx 1.2.0 has requirement jsonschema<=4.17.3, but you'll have jsonschema 4.22.0 which is incompatible.
```
so, I apply this restriction

Reply:
it seems to be ok